docs/product/explore/feature-flags/index.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: "Feature Flags"
+sidebar_order: 100
+description: "Learn how to set up and interact with Sentry's feature flag evaluation tracking and feature flag change tracking."
+---
+
+<Alert level="info" title="Currently in Beta">
+
+The support for **feature flag change tracking** and **feature flag evaluation tracking** is currently in beta.
```

Comment:
Update alert to be the same as copy suggested in https://github.com/getsentry/sentry-docs/pull/11971